_docs/_latest/setup/manual.md

```diff
@@ -56,7 +56,7 @@ service account, see
 
 * For information about the specifications needed for a Forseti server
   VM instance, see the
-  [Compute Engine deployment template](https://github.com/forseti-security/forseti-security/blob/master/deployment-templates/compute-engine/server/forseti-instance-server.py)
+  [Compute Engine deployment template](https://github.com/forseti-security/forseti-security/blob/master-old/deployment-templates/compute-engine/server/forseti-instance-server.py)
```

Comment:
The deployment templates were removed as part of removing support for the Python installer/DM. I have updated these broken links to use the `master-old` branch which points to the commit that `master` was pointing to before it was updated.